nsist/pypi.py

```diff
@@ -181,7 +182,15 @@ def extract_wheel(whl_file, target_dir):
     for p in td.iterdir():
         if p.suffix not in {'.data', '.dist-info'}:
             if p.is_dir():
-                shutil.copytree(str(p), str(target / p.name))
+                try:
+                    shutil.copytree(str(p), str(target / p.name))
+                except OSError as e:
```

Comment:
If the distutils behaviour is what we want in any case, does it make sense to call that directly rather than trying `shutil` first?